chore: Add warning about `table_options` removal (#13196)
cloudquery/cloudquery: website/pages/docs/plugins/sources/aws/configuration.mdx

  When set to `true` plugin will sync data from APIs that incur a fee. 
  Currently only `aws_costexplorer*` and `aws_alpha_cloudwatch_metric*` tables require this flag to be set to `true`.

<Callout type="warn">

> The `table_options` parameter will be deprecated and removed in a future release. Please reach out to CloudQuery team if you need this feature

Reviewer: ```suggestion
The `table_options` parameter is deprecated and will be removed soon. Please reach out to the CloudQuery team if you require this feature
```
